azure-pipelines.yml

```diff
@@ -119,6 +119,19 @@ jobs:
     # Only run on pull requests to check new commits only
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check commit metadata
+  - bash: |
+      find . -name '*.vendor.hjson' \
+        | grep -v /vendor/lowrisc_ibex/vendor/ \
+        | xargs -n1 util/vendor.py --check-patches --verbose \
+        && git diff --exit-code
+      if [[ $? != 0 ]]; then
+        echo -n "##vso[task.logissue type=error]"
+        echo "Vendored repositories not up-to-date. Re-clone them with util/vendor.py."
+        exit 1
+      fi
+    condition: always()
+    continueOnError: true
+    displayName: Check Vendored Repos Match Lockfiles
```

Comment:
nit: we don't typically for for headline style capitalization in these messages.

Reply:
yeah, would be more consistent here. But certainly nothing to worry about for too long :smile: